src/main/java/com/google/api/codegen/discovery/config/java/JavaTypeNameGenerator.java

```diff
@@ -61,12 +62,23 @@ public String getSubpackage(boolean isRequest) {
   }
 
   @Override
-  public String formatValue(String value, Field.Kind kind) {
-    switch (kind) {
-      case TYPE_STRING:
-        return "\"" + value + "\"";
-      default:
-        throw new IllegalArgumentException("unsupported kind: " + kind.toString());
+  public String getExample(String format) {
```

Comment:
Got it, I've split this into two functions:
`getStringFormatExample` and `getFieldPatternExample`
Those functions only return the language specific string/description with the appropriate quotes.

`"ex: "` is prefixed in the `createFieldInfo` function in `ApiaryConfigToSampleConfigConverter` if there's a field pattern, otherwise there is no prefix.